nixos/modules/virtualisation/parallels-guest.nix

```diff
@@ -89,5 +89,58 @@ in
       };
     };
 
+    systemd.services.prlcc = {
+      description = "Parallels Control Center";
+      wantedBy = [ "graphical.target" ];
+      serviceConfig = {
+        ExecStart = "${prl-tools}/bin/prlcc";
```

Comment:
I am not familiar at all with parallels to excuse my question:
What does `prlcc` that it is started both as a global service and a user in a graphical session?

Reply:
prlcc is a general daemon which handles mouse/dynamic resolution/communication with host/shared folders. I put it as a "global" to try and get it working on the login screen; it can be removed. The user service is the important one which works perfectly once logged in.

I don't recall seeing prldhd in prl-12, so I'm assuming they merged its functionality into prlcc

Reply:
Is there any recommendation by upstream? Running `x11` applications as root can create security problems because their is not user separation in x11.

Reply:
@Mic92 Fixed! I removed prlcc as a global service, and everything still works.